docs/style-guidelines.md

```diff
@@ -17,4 +17,96 @@ To install it go to the root of your explorer repository and run the following c
 
 
 ### Rider
-You can find a settings export file in the root of the project called "rider_codeStyleDCLSetting". Bear in mind that the conversion between VS and Rider is not 1 on 1 but it's good enough.
\ No newline at end of file
+You can find a settings export file in the root of the project called "rider_codeStyleDCLSetting". Bear in mind that the conversion between VS and Rider is not 1 on 1 but it's good enough.
+
+# Code conventions
+[Microsoft’s guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/) are used as a baseline. Here you will find short summary, highlights and specifics related to Unity, which we suggest to follow.
+### Naming conventions
+Use
+* `PascalCase` - Namespace, Class, Struct, Interface, Enumeration and its Enumerators, Method, Delegate declaration, Event, Property and public Field;
+* `camelCase` - Field, local Variable, methods Parameter;
+* `CAPITALS_SNAKE_CASE` - Constants;
+* `I` prefix in front of Interface name;
+```csharp
+namespace MyProject                                     // Namespace -> PascalCase
+{
+    public enum Side { Left, Right }                    // Enumeration and Enumerators -> PascalCase 
+    public delegate void Change<T> (T current);         // Delegate declaration -> PascalCase
+    public interface IInitializable { }                 // Interface -> PascalCase, starts with 'I'
+    
+    public class MyClass : IInitializable               // Class/Struct -> PascalCase.
+    {
+        public const string ASSET_PATH = "AvatarUI";    // Constant -> CAPITALS_SNAKE_CASE 
```

Comment:
guys, what do you think also about the `constants`? I would rather go with the .NET standard and just use `PascalCase` for it instead of `CAPITALS_SNAKE_CASE`, or even go with Google C# standards where 
> _"Naming convention is unaffected by modifiers such as const, static, read-only, etc."_
My point here would be similar to @AjimenezDCL, that IDE is so powerful nowadays and makes constants always Bold, so it is easily distinguishable.